ui/packages/components/src/RunDetailsV2/RunDetails.tsx

```diff
@@ -95,22 +97,24 @@ export function RunDetails(props: Props) {
       <div className="flex gap-4">
         <div className="grow">
           <div className="ml-8">
-            {runRes.error || resultRes.error ? (
+            <RunInfo
+              cancelRun={cancelRun}
+              className="mb-4"
+              pathCreator={pathCreator}
+              rerun={rerun}
+              initialRunData={props.initialRunData}
+              run={nullishToLazy(run)}
+              runID={runID}
+              standalone={standalone}
+              result={resultRes.data}
+            />
+            {props.initialRunData?.status !== 'QUEUED' && (runRes.error || resultRes.error) ? (
```

Comment:
@djfarrelly I think we need to suppress the error here as well when we're in queued state because the run info card will show the loader internally but we'll still show an error here.